src/Omnipay/Realex/Message/EnrolmentRequest.php

```diff
@@ -120,8 +130,22 @@ protected function createResponse($data)
         return $response;
     }
 
+//    public function getEndpoint()
```

Comment:
Commented-out code needs removing or a comment explaining why this is a temporary action adding.